docs/pip.md

```diff
@@ -266,6 +266,42 @@ If you do need to use a git url, Cachi2 requires that it specifies a full commit
 git urls. Cachi2 requires hashes for https. Using one --hash makes pip require hashes for everything. Pip does not
 support hashes for git dependencies. Please use https urls instead.*
 
+### Supported options
+
+Requirements files support some `pip install` options - see
+<https://pip.pypa.io/en/stable/reference/requirements-file-format/#supported-options>.
+
+Cachi2 supports a small subset of them, ignores those that are not relevant for prefetching, and raises an error for
+those that *are* relevant but aren't supported.
+
+#### Global
+
+##### [`--index-url`](https://pip.pypa.io/en/stable/cli/pip_install/#install-index-url)
+
+Make Cachi2 download packages from the specified Python Package Index server.
+
+Note: applies to all the packages (and only the packages) from the file which contains the `--index-url` option.
+If file A contains `--index-url` and file B does not, Cachi2 will download the packages declared in B from the default
+index server (<https://pypi.org/simple/>).
+
+:warning: Do not include credentials in the index url. If needed, provide authentication via a `.netrc` file:
+<https://pip.pypa.io/en/stable/topics/authentication/#netrc-support>
```

Comment:
nitpick: do you think this very warning deserves the bold markup tags? After all, it has serious security implications.

Reply:
Good idea, added